packages/docs-reanimated/src/examples/MakeMutable.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+  Text,
+  StyleSheet,
+  View,
+  TouchableOpacity,
+  useColorScheme,
+} from 'react-native';
+
+import React, { useCallback, useMemo, useState } from 'react';
+import Animated, {
+  makeMutable,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+import type { SharedValue } from 'react-native-reanimated';
+
+type CheckListSelectorProps = {
+  items: string[];
+  onSubmit: (selectedItems: string[]) => void;
+};
+
+function CheckListSelector({ items, onSubmit }: CheckListSelectorProps) {
+  const checkListItemProps = useMemo(
+    () =>
+      items.map((item) => ({
+        item,
+        // highlight-next-line
+        selected: makeMutable(false),
+      })),
+    [items]
+  );
+
+  const handleSubmit = useCallback(() => {
+    const selectedItems = checkListItemProps
+      .filter((props) => props.selected.value)
+      .map((props) => props.item);
+
+    onSubmit(selectedItems);
```

Comment:
```suggestion
    runOnUI(() => {
      const selectedItems = checkListItemProps
        .filter((props) => props.selected.value)
        .map((props) => props.item);

      runOnJS(onSubmit)(selectedItems);
    })()
```

Probably would have to change to this implementation to avoid reading `.value` on the JS thread.